src/main/java/uk/gov/justice/hmpps/datacompliance/web/dto/UalOffenderResponse.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.justice.hmpps.datacompliance.web.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@ApiModel(description = "Ual Offender data")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(NON_NULL)
+public class UalOffenderResponse {
+
+    @ApiModelProperty(required = true, value = "This is known as the 'prison number' or 'offender booking number' in Nomis")
```

Comment:
Is offender *booking* number right? Isn't it just offender number? The number of names HMPPS use for all these is crazy isn't it! 🤯

Reply:
Sorry that's a mistake will update :)